webpack/assets/javascripts/react_app/redux/middlewares/IntervalMiddleware/IntervalActions.js

```diff
@@ -2,17 +2,11 @@ import { STOP_INTERVAL, START_INTERVAL } from './IntervalConstants';
 
 export const stopInterval = key => ({
   type: STOP_INTERVAL,
-  payload: {
-    key,
-  },
+  key,
 });
 
-export const startInterval = (key, callback, interval, ...intervalArgs) => ({
+export const startIntervalAction = (key, intervalID) => ({
```

Comment:
I don't think "Action" needs to be part of the name.

Reply:
When it's called inside the middleware it may be confusing since we have `setInterval`.. though I could import it as another name

Reply:
@laviro I'd be in favor of calling it `startInterval` here and then importing as a different name if you need to avoid confusion elsewhere.  Just seems redundant to name an action creator `__Action`.